soc/nxp/imxrt/CMakeLists.txt

```diff
@@ -37,6 +37,24 @@ if(CONFIG_SOC_SERIES_IMXRT10XX OR CONFIG_SOC_SERIES_IMXRT11XX)
   zephyr_compile_definitions(XIP_EXTERNAL_FLASH)
 endif()
 
+if(CONFIG_SOC_SERIES_IMXRT118X)
```

Comment:
Where should we document this stuff? Is there a link to some NXP documents on the various sections?